doc/src/manual/running-external-programs.md

```diff
@@ -60,6 +60,40 @@ julia> open(`less`, "w", STDOUT) do io
 3
 ```
 
+## Cmd Object
+Normally, to create a `Cmd` object in the first place, one uses backticks, e.g.
+
+```jldoctest
+julia> Cmd(`echo "Hello world"`, ignorestatus=true, detach=false)
+`echo 'Hello world'`
+```
+
+There are some keyword arguments available to customize the `Cmd` object usage. You can set the working directory for the command by using the `dir` kwarg, e.g.
+
+```jldoctest
+julia> Cmd(`pwd`, dir ="..")
+setenv(`pwd`; dir="..")
+```
+
+You can use [`setenv`](@ref) function to set the environment variables to use while running the given command, e.g.
+
+```jldoctest
+julia> setenv(`echo "home"`,"JL_DOC"=>"julia_docs")
+setenv(`echo home`,String["JL_DOC=julia_docs"])
+```
+
+To execute the given command in an environment that is temporarily modified (not replaced as in `setenv`) by zero or more "var"=>val arguments kv you can use [`withenv`](@ref), e.g.
```

Comment:
`given` -> `a`? Not sure what is given here.

Should `"var" => val` be code highlighted?

The `kv` seems out of place.